hangman.rb

```diff
@@ -0,0 +1,202 @@
+# Wang & I finished the  breakfast & lunch requirements & part of dinner together.
+# We weren't sure we'd have time to finish the dinner requirements, so we focused
+# on not penalizing the user for guessing the same letter twice & at the same time
+# handling guesses that weren't qualified letters. We did not add difficulty levels.
+
+# This morning, I also added a couple lines to handle guessing the whole word, but
+# it's not a fully implemented feature: it doesn't add a piece to the hangman for
+# bad guesses. But it does let you skip ahead to winning if you already know the
+# word, which is what I was going for. It also throws some errors. >_>
+
+# -*- coding: utf-8 -*-
+require "colorize"
+
+# pair project w/ Wang :)
+
+class Hangman
+# 1. {0}
+# 2. (\
+# 3. /)
+# 4. ~
+# 5. (:)
+# 6. _/
+# 7. \_
+
+  def initialize
+    # hangman board:
+    @first        = "     ______       "
+    @second       = "    |      |      "
+    @third        = "           |      "
+    @fourth       = "           |      "
+    @fifth        = "           |      "
+    @sixth        = "           |      "
+    @seventh      = "    _______|___   "
+    @eighth       = "   |           |  "
+    @ninth        = "                  "
+    @tenth        = "  H A N G M A N   " # change based on how many letters, etc
+
+    # start game
+    start
+  end
+
+
+  def display_board
+    @hangman = [@first, @second, @third, @fourth, @fifth,
+                @sixth, @seventh, @eighth, @ninth, @tenth]
+
+
+    @hangman.each do |line|
+      puts line
+    end
+
+    puts "\n"
+  end
+
+
+  def set_answer
+    possible_answers = [ "potato", "broccoli", "animal", "hotdog",
+      "elephant", "cat", "taco", "whale", "walrus", "vulture", "lunch",
+      "morning", "evening", "scotch", "whiskey", "salad", "notebook",
+      "laptop", "coffee", "caffeine", "solar", "system", "planet",
+      "cheerful", "happy", "emotion", "below", "scream", "shout", "jumprope",
+      "wizard", "rainfall", "cloudy", "sunlight", "sunshine", "program",
+      "hangman", "sticky", "oranges", "eraser", "pencil", "powerful",
+      "creative", "cleaning", "nontoxic", "keyboard", "key", "any", "four",
+      "badge", "ridge", "cringe", "fringe", "lawful", "lawless", "chaotic",
+      "chaos", "crafty", "space", "outside", "outdoor", "cabin", "camping",
+      "tentacle", "monster", "you", "our", "hour"
+      ]
+
+    # select random word
+    which_random_index = rand(0...possible_answers.length)
+    @answer = possible_answers[which_random_index]
```

Comment:
You could also use the `sample` method which will retrieve a "random" element from the possible_answers